packages/block-editor/src/components/roving-tab-index/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import RovingTabIndexContainer from './container';
+import RovingTabIndexItem from './item';
+
+/**
+ * @see https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/roving-tab-index/README.md
+ */
+const RovingTabIndex = {
```

Comment:
Not sure whether this pattern of grouping components under an object is considered good (e.g `RovingTabIndex.Container`, `RovingTabIndex.Item`). They could just be exported separately (`RovingTabIndexContainer` ...).

Reply:
I believe the current approach is to avoid doing that because it prevents dead code elimination for 3rd party projects.

Reply:
Ah, ok, good to know! I shall change that.